### ISSUE TITLE
Breaks syntax highlighting of non angular or angular html files having @import in style

### ISSUE THREAD

**Issue by imaksp** [angular/vscode-ng-language-service#1979]
# 🐞 bug report

### Is this a regression?

Yes

### Description

If non angular html file or angular index html file has style tag with @import url statement it breaks syntax highlighting of rest of the file.

## Bug Type
What does this bug affect

- [x] Angular Language Service VSCode extension
- [ ] Angular Language Service server

## Reproduction

1.  add this style tag following in any html file & check syntax highlighting of rest of the content.
```html
  <style>
    @import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;600&display=swap');
  </style>
```

**Expected behavior**

It should not break normal html syntax highlighting.

## Screenshots
<img width="909" alt="syntax-h" src="https://github.com/angular/vscode-ng-language-service/assets/2557058/9d7cfbed-0822-4c84-b82f-6042cd7676cb">

## 🌍  Your Environment
**Angular Version:**
```
17.0.5
```

Extension Version:
```
17.0.2
```
VSCode Version:
```
Version: 1.84.2
Commit: 1a5daa3a0231a0fbba4f14db7ec463cf99d7768e
Date: 2023-11-09T10:52:57.054Z
Electron: 25.9.2
ElectronBuildId: 24603566
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 23.1.0
```

Operating System:
```
Mac OS 14.1
```

**Comment by atscott** [comment]
This particular issue will be fixed by https://github.com/angular/vscode-ng-language-service/pull/1952

As for syntax highlighting applying to non-Angular templates, unfortunately there is no way to dynamically enable or disable syntax highlighting in VSCode. https://github.com/microsoft/vscode/issues/68647

**Comment by imaksp** [comment]
Ok, will it be possible to add config based on file extension, like Angular syntax highlighting only gets applied if file ends with `component.html` OR `ng.html`?

**Comment by atscott** [comment]
Closing as resolved by #1952 which was included in the release today.
